numpy/lib/function_base.py

```diff
@@ -4326,6 +4327,69 @@ def meshgrid(*xi, **kwargs):
             return np.broadcast_arrays(*output)
 
 
+def meshgridify(*xi, **kwargs):
```

Comment:
Hi, thanks for your comments.
I just sent out a subscription request for the mailing list. I will post my pull request there.
Regarding the place of the function, I thought near the meshgrid function, can't be too wrong.
I'm open for any suggestions regarding the name ;-)